packages/etherlime-zos-deployer/src/zos-deployer/zos-jsonrpc-private-key-deployer.ts

```diff
@@ -0,0 +1,120 @@
+import { ZWeb3, Contract, Contracts, SimpleProject } from 'zos-lib';
+import * as fs from 'fs';
+import * as path from 'path';
+
+import { JSONRPCPrivateKeyDeployer } from 'etherlime-lib';
+import { TxParams, CompiledContract } from 'etherlime-lib';
+import { ProxyData } from './../types'
+import { BigNumber, bigNumberify } from 'ethers/utils';
```

Comment:
Could we import here only the utils from ethers:
`import { utils } from 'ethers';`
 and then use it in the code as: 
`utils.BigNumber; utils. bigNumberify`;
This is needed because of the using of etherlime in front-end frameworks.